functions/packages.bash

```diff
@@ -798,3 +798,160 @@ setup_evcc() {
   echo -n "$(timestamp) [openHABian] Created EVCC config, restarting ... "
   if cond_redirect systemctl restart evcc.service; then echo "OK"; else echo "FAILED"; fi
 }
+
+## Function for (un)installing ESPhome dashboard
+## The function must be invoked UNATTENDED.
+## Valid arguments: "install" or "remove"
+##
+##   install_esphomedashboard(String action)
+##
+##
+install_esphomedashboard() {
+  local port=6052
+  local installText="This will install ESPhome dashboard\\nUse the web interface on port $port to access ESPhome dashboard web interface."
+  local removeText="This will remove ESPhome dashboard"
+  
+  if [[ $1 == "remove" ]]; then
+    if [[ -n $INTERACTIVE ]]; then
+      whiptail --title "ESPhome dashboard removal" --msgbox "$removeText" 7 80
+    fi
+    echo -n "$(timestamp) [openHABian] Starting ESPHome Dashboard uninstallation..."
+
+    # Function to check the success of the last executed command
+    check_success() {
+        if [ $? -ne 0 ]; then
+            echo -n "$(timestamp) [openHABian] Error: $1"
+            exit 1
```

Comment:
Lets not exit, use return instead, we don't want to hard exit the script, but if we return the main script will handle it gracefully.

Also, for consistency, it would be best to use the same style of failing used elsewhere in the codebase. I like the thought process behind this style of failing, but it would be better to introduce it in the future at a project level.

Reply:
done